internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -306,6 +308,16 @@ func (r *ClusterDeploymentsReconciler) createNewCluster(
 		UserManagedNetworking: swag.Bool(isUserManagedNetwork(cluster)),
 	}
 
+	installEnv, err := getInstallEnvByClusterDeployment(ctx, r.Client, cluster)
+	if err == nil && installEnv != nil {
+		if len(installEnv.Spec.AdditionalNTPSources) > 0 {
```

Comment:
Need to notify installenv here as well no?

Reply:
Yes. Done.